doc/community/developer-guide.md

```diff
@@ -353,6 +353,14 @@ To run some of the tests:
 pytest src/tests/integration/test_generic_db_operations.py::test_profile_query
 ```
 
+```{important}
+There are tests which check compatibility of  Microsoft SQL Server. In order to run them you you will need to install the ODBC driver for SQL Server (https://learn.microsoft.com/en-us/sql/connect/odbc/microsoft-odbc-driver-for-sql-server?view=sql-server-ver16). 
+
+Instructions for Linux can be found at https://learn.microsoft.com/en-us/sql/connect/odbc/microsoft-odbc-driver-for-sql-server?view=sql-server-ver16
+Instructions for MAC can be found at https://learn.microsoft.com/en-us/sql/connect/odbc/linux-mac/install-microsoft-odbc-driver-sql-server-macos?view=sql-server-ver16
+```
```

Comment:
Some typos:
1. compatibility of  Microsoft SQL Server -> compatibility of Microsoft SQL Server
2. them you you will need to -> them you will need to
3. MAC -> Mac

Also, To make it easier to read and navigate, can you please list the instructions for Mac and Linux (can we add Windows as well?) and instead of using the entire URL, can we name it? (i.e. `[name](https://....)`)

Reply:
I will take care of this , in the evening!

Reply:
I made a push with the changes you suggested for the  developer guide, and no entries to change-log. The PR got  closed imemdiately.

Reply:
hey @axd353, no worries. I closed this since we finished evaluating the assignment. no need to do any follow ups here. @yafimvo will email you with details for the next interview stage